routes/web.php

```diff
@@ -27,9 +28,9 @@
     Route::resource('users', UserController::class)->only(['store', 'index', 'destroy', 'update']);
 
     Route::resource('roles',\App\Http\Controllers\RoleController::class)->except(['show', 'edit']);
-
     Route::resource('rooms',\App\Http\Controllers\RoomController::class)->only(['store', 'index', 'update', 'destroy']);
-    Route::resource('rooms',\App\Http\Controllers\RoomController::class)->only(['store', 'index', 'update','destroy']);
+    Route::put('room/restrictions/{id}', RestrictionsController::class.'@update')
```

Comment:
I believe the approach is now to pass an array of `[controller::class, 'method_name']`